### ISSUE TITLE
FNX-5053 ⁃ TalkBack highlighter overlaps browser user interface

### ISSUE THREAD

**Issue by asadotzler** [mozilla-mobile/fenix#1472]
## Steps to reproduce
Enable TalkBack
Surf to a web page
swipe until a paragraph is selected near the bottom of the page

### Expected behavior
The TalkBack visual selection indicator should not overlap the browser UI at the bottom of the page

### Actual behavior
The TalkBack visual selection indicator overlaps the browser UI at the bottom of the page. 

### Device information
* Android device: ? Samsung Galaxy S6 
* Fenix version: ? 2019-04-03

**Comment by asadotzler** [comment]
I'm unable to reproduce this in the latest build.
